extension/WebJobs.Extensions.RabbitMQ.Samples/RabbitMQSamples.cs

```diff
@@ -120,14 +120,9 @@ public static class RabbitMQSamples
 
         public class TestClass
         {
-            private readonly int _x;
-            private readonly int _y;
+            public int X { get; set; }
 
```

Comment:
Private fields are ignored during serialisation and de-serialisation. Serializing TestClass object was yielding an empty JSON object `{}`.